### ISSUE TITLE
Meta Description Missing From LTS "ghost_head" Helper

### ISSUE THREAD

**Issue by triestpa** [TryGhost/Ghost#8713]
### Issue Summary

One of the suggested changes for migrating themes to Ghost 1.0 is removing the meta description from `default.hbs`.  

In Ghost 1.0, this description is instead added in the `{{ghost_head}}` helper - https://github.com/TryGhost/Ghost/blob/master/core/server/helpers/ghost_head.js#L109

In the Ghost LTS branch, this addition is missing -
https://github.com/TryGhost/Ghost/blob/lts/core/server/helpers/ghost_head.js#L95

Unfortunately, this means that any themes that are updated to full 1.0 compatibility are now missing their "description" meta when rendered on the LTS version.  

### Solution
Since the latest LTS version is meant to provide forward compatibility with 1.0 themes, the meta description should be added to the `{{ghost_head}}` helper.

**Comment by AileenCGN** [comment]
Hey @triestpa!

Yes, this is one thing we're aware of. The problem with your suggested solution is, that for users who don't migrate their theme to 1.0 compatibility, the `meta-description` would be outputted twice, if we'd include it in `{{ghost_head}}` for LTS.

The usage of the `{{meta_description}}` in HTML `<head>` is deprecated in Ghost 1.0, but will not cause a fatal error, which means it'll still possible to upload and activate a it t Ghost 1.0.

tl;dr:
- If Ghost includes `meta-description` in the `{{ghost_head}}` helper for LTS, it'll be outputted twice for themes that are not migrated to 1.0.
- If it's removed in a migrated theme, because it'll be deprecated in Ghost 1.0, there'll be no `meta-description` in LTS versions.
- If it's left in the migrated theme, it'll be fine in LTS versions but outputted twice on Ghost 1.0.

@ErisDS What are your thoughts on that?

**Comment by ErisDS** [comment]
@AileenCGN I agree with your assessment. 

Ghost LTS is forwards compatible in so much as 1.0.0 themes will not cause fatal errors, so as to ease the migration process. That is - it is now _possible_ to upload a 1.0.0 theme to LTS and it will not cause errors, whereas previously it would have.  

There is no good solution for making the meta description back or forwards compatible, therefore the change happens in a major version bump. This is correct behaviour IMO.

**Comment by triestpa** [comment]
Ok fair enough, thanks for the clarification!
